module-4/backend/week_three.md

```diff
@@ -5,15 +5,47 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+
+node is the backend for javascript
+
 2. What is Express? What is Express similar to in the Ruby world?
+
+express is a framwork for node. and it is similar to rails in the ruby world.
```

Comment:
It is the framework, more similar to Sinatra, though - because of it's lack of opinions and how lightweight it is.